extensions/amp-ad-exit/0.1/config.js

```diff
@@ -121,6 +134,21 @@ function assertTarget(name, target, config) {
       user().assert(
           pattern.test(variable), '\'%s\' must match the pattern \'%s\'',
           variable, pattern);
+      const vendor = target.vars[variable]['vendorAnalyticsSource'];
+      if (vendor !== undefined) {
```

Comment:
(Undone, because Curt pointed out that it might be possible for some vendor's name to be falsey)